packages/fetchai/skills/echo/dialogues.py

```diff
@@ -60,6 +60,6 @@ def role_from_first_message(  # pylint: disable=unused-argument
 
         BaseDefaultDialogues.__init__(
             self,
-            self_address=self.context.agent_name,
+            self_address=self.context.agent_address,
```

Comment:
Revert please, this is intentional!

Reply:
Sure. Why is it?

Reply:
For the echo example in Quickstart to work easily. Otherwise you first need to get the address of the agent to send it a message.